README.md

```diff
@@ -292,6 +290,50 @@ import hljs from 'highlight.js';
 import 'highlight.js/styles/github.css';
 ```
 
+### Browser ES6 Modules
+
+*Note: You need to install `@highlightjs/cdn-assets` package instead of `highlight.js`. See [Download prebuilt CDN assets](#download-prebuilt-cdn-assets)*
+
+To import the library and register only those languages that you need:
+
+```js
+import hljs from './node_modules/@highlightjs/cdn-assets/es/core.js';
+import javascript from './node_modules/@highlightjs/cdn-assets/es/languages/javascript.min.js';
```

Comment:
Isn't this assuming a lot about someone's file structure?  I feel like publishing `node_modules` onto a public site is a bit of a smell, is that normal these days?

Reply:
Ah you addressed that below... I'll mull on this.

Reply:
Thanks.

First of all, use of web technologies (HTML, CSS, JS) is no longer limited to website/webapp. With the help of Electron, web technologies can used in desktop app as well. This means importing from `node_modules` is no longer a concern in desktop app.

Now, about the public site, you are right that it is not normal (or lets say, not common in practice) to publish `node_modules`. I think it is because of lack of awareness about different ways npm packages can be installed in a project. Most of people think that `package.json` stays at the root directory and all packages should be installed at the root directory. However, we can create separate `package.json` at public directory for browser specific packages.

```
/
/package.json          (may contain server-specific packages like Express, or may only contain devDependencies like Webpack in case the app is not using Node for server (for e.g. PHP app))
/node_modules/
/public/package.json          (contains browser-specific packages like highlight.js, bootstrap)
/public/node_modules/
/public/index.html
```

This way, packages at top level and any other server specific code does not get exposed to the public.

There is a change of security issue if a npm package contain file that can be directly executed by server (for e.g. PHP file). In that case, a CVE should be created.

Lastly, we can generate different code for the `prod` environment. Tools like [Vite](https://vitejs.dev/guide/build) may be helpful regarding that.  This way, we can use original code for development and generated code for production.

I can also update the doc to include warning:
```
Warning: publishing `node_modules` can lead to security problem. Use with caution.
```

If you think it is not a good idea at all to have `node_modules` being publicly exposed, I can update the doc to manually copy code from `@highlightjs/cdn-assets` package and paste in public directory (or create symlink).

Let me know what you think.